app/mentions/mentions.test.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+var expect 		= require('chai').expect;
+var should 		= require('chai').should();
```

Comment:
Use expect or should, but not both. We have been using expect everywhere else in the project